src/safeds_stubgen/api_analyzer/_ast_visitor.py

```diff
@@ -95,13 +95,22 @@ def enter_moduledef(self, node: MypyFile) -> None:
             elif isinstance(definition, ExpressionStmt) and isinstance(definition.expr, StrExpr):
                 docstring = definition.expr.value
 
+        # Create module id to get the full path
+        module_id_parts = node.fullname.split(self.api.package)[1:]
+        module_id = self.api.package.join(module_id_parts)
```

Comment:
This seems wrong: `"a".join(["b", "c"])` produces `"abac"`. The `split` in the line above also looks fishy.

Reply:
I don't quite understand the problem here, could you elaborate it a bit further? I added code comments if the code was unclear.